### PR TITLE
Fix checkboxes in ldap settings

### LINKED ISSUE #19238: Checkboxes missing in LDAP wizard
### Steps to reproduce
1. Enable LDAP user backend
2. Setup a valid configuration

### Expected behaviour
The "login attributes" tab shows a clear way to enable / disable login with the LDAP username or email.

### Actual behaviour
There is no checkbox to show if those attributes are enabled or not, and the only way to modify it is by clicking in the text. I find it very confusing because clicking in the text isn't common, and also we cannot know the state

### Server configuration
**Operating system**: centos 6.6

**Web server:** httpd

**Database:** mysql

**PHP version:** 5.4

**ownCloud version:** 8.2 beta 1

**Updated from an older ownCloud or fresh install:** fresh install

screenshot from 2015-09-22 09 59 50

### PR DESCRIPTION
**Pull request by Henni**
fixes #19238 

I'm not able to test this because I have no ldap backend.
So please take a look @jvillafanez @PVince81

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
@blizzz

**Comment by blizzz** [comment]
@Henni it fixes it, however the Checkboxes now are on the wrong side:

screenshot

In this case they should be right of the label. Can you adjust that?

**Comment by Henni** [comment]
@blizzz please take another look. should be fixed now

**Comment by blizzz** [comment]
@Henni better:

y a a y 4

Is it possible to  left align them?

And a general note (out of scope): enabled and disabled checkboxes look different, but the difference is really small and hard to spot imho:

y a a y 4 vs. 
y a a y 4

**Comment by Henni** [comment]
As said before I can't reproduce this.
Maybe @jancborchardt can help out.

Also what do you think about the disabled state of the checkboxes, @jancborchardt ?

**Comment by jancborchardt** [comment]
The checkboxes should be moved to the left (to be right to the text), not fly in the middle like that. :)

@Henni maybe the disabled state needs less opacity. Currently 60%, try 50, or 35.

**Comment by Henni** [comment]
@jancborchardt I don't know why the checkboxes are centered because I'm not able to test it. Maybe you can take a look at this and fix it.

**Comment by blizzz** [comment]
There's a script for starting an ldap docker container, if you want to try it out: https://github.com/owncloud/administration/tree/master/ldap-testing

File: apps/user_ldap/templates/part.wizard-loginfilter.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 			<input type="checkbox" id="ldap_loginfilter_username"
 				   title="<?php p($l->t('Allows login against the LDAP / AD username, which is either uid or samaccountname and will be detected.'));?>"
 				   name="ldap_loginfilter_username" value="1" />
+			<label for="ldap_loginfilter_username"></label>
```
Comment (MorrisJobke) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: This will duplicate the label :confused: It's already there (lines 7-9)

**Comment by MorrisJobke** [comment]
Since the original issue causing PR was reverted I would like to close this one and fix this together with the switch to the new checkbox style.

 @Henni @blizzz Is this okay?

**Comment by Henni** [comment]
@MorrisJobke :+1:

**Pull request closed without merging**